gitlab.go

```diff
@@ -187,9 +187,6 @@ func (c *Client) SetBaseURL(urlStr string) error {
 		return err
 	}
 
-	// Set the encoded opaque data
-	c.baseURL.Opaque = c.baseURL.Path
-
```

Comment:
As you wish, I can revert all changes related to 1.5.

But are there any known incompatibilities or big changes between 1.4 and 1.5?
1.5 is out for more than 3 months and some of my favorite projects written in go like docker and drone already use 1.5.x.
So I don't think It should be a big problem for another software using this package to upgrade to 1.5 if they want to use this package ... Additionally they will benefit from Go 1.5 improvements (garbage collector related, multicore support in scheduler external dependencies vendoring, ..)    :stuck_out_tongue_winking_eye: :sunglasses:

Let me know if you change your mind.  :smile:  I would like to work on it today evening (CET).